src/App.svelte

```diff
@@ -16,6 +16,7 @@
   import Header from '@app/Header.svelte';
   import Loading from '@app/Loading.svelte';
   import Modal from '@app/Modal.svelte';
+  import LinearGradient from "@app/base/projects/LinearGradient.svelte";
```

Comment:
If this is to be used as a global gradient, then maybe move it to `@app/LinearGradient.svelte`?